CHANGELOG.md

```diff
@@ -6,8 +6,9 @@ All notable changes to this project will be documented in this file.
 
 ### New Features
 ### Bug Fixes
-* Remove explicit version on `micrometer-registry-prometheus` dependency. (#563)
-* Send a `TaskNotificationType` to worker through a 2XX HTTP status code. (#564)
+- Remove explicit version on `micrometer-registry-prometheus` dependency. (#563)
+- Send a `TaskNotificationType` to worker through a 2XX HTTP status code. (#564)
+- Add missing info on to log (#566)
```

Comment:
very minor, I think there is a glitch with on/to
I like title of your PR
```suggestion
- Improve log on `canUpdateReplicateStatus` method. (#566)
```
Go to quality section?